FreeRDP/FreeRDP
# Rdpsettings script

This PR provides the script scripts/update-rdpSettings.
(Since this scripts outputs the #define in a fixed format, there's a first commit reformatting those defines, so that future applications of the scripts only modify the relevant lines).

* Give instructions on how to test your changes
The path of the file include/freerdp/settings.h  is hardcoded in the script.

1. Edit this file, adding or modifying fields in the struct rdp_settings;
2. Save the file.
3. cd to the FreeRDP repository directory (where .git is).
4. run the script:  ./scripts/update-rdpSettings
5. reload the file include/freerdp/settings.h and see the updated \#defines.
